packages/react/.storybook/_container.scss

```diff
@@ -1,9 +1,108 @@
 $css--font-face: true;
+$css--helpers: true;
+$css--body: true;
+$css--use-layer: true;
 $css--reset: true;
+$css--plex: true;
 
 $prefix: 'bx';
 
-// IMPORTANT: This import path should _not_ be used outside our source tree
-// as `src` directory is _not_ meant to be shipped in our NPM package.
-// Use e.g. `@import '~carbon-components/scss/globals/scss/styles.scss'` instead.
-@import '~carbon-components/src/globals/scss/styles.scss'; // SEE THE NOTE ABOVE
+@import '~carbon-components/src/globals/scss/feature-flags';
+@import '~carbon-components/src/globals/scss/vars';
+@import '~carbon-components/src/globals/scss/colors';
+@import '~carbon-components/src/globals/scss/theme';
+@import '~carbon-components/src/globals/scss/mixins';
+@import '~carbon-components/src/globals/scss/layout';
+@import '~carbon-components/src/globals/scss/layer';
+@import '~carbon-components/src/globals/scss/spacing';
+@import '~carbon-components/src/globals/scss/typography';
+@import '~carbon-components/src/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
+@import '~carbon-components/src/globals/scss/css--reset';
+@import '~carbon-components/src/globals/scss/css--font-face';
+@import '~carbon-components/src/globals/scss/css--helpers';
+@import '~carbon-components/src/globals/scss/css--body';
+@import '~carbon-components/src/globals/grid/grid';
+
+$accordion-type-tokens: map-merge(
+  $tokens,
+  (
+    'body-long-01': $body-long-02,
+  )
+);
+$accordion-heading-height: carbon--mini-units(6);
+@import '~carbon-components/src/components/accordion/mixins';
+@include carbon--type-theme($accordion-type-tokens) {
+  @include exports('accordion') {
+    @include accordion;
+  }
+}
+
+$button-type-tokens: map-merge(
+  $tokens,
+  (
+    'body-short-01': $body-short-02,
+  )
+);
+$button-field-height: carbon--mini-units(6);
+@import '~carbon-components/src/components/button/mixins';
+@include carbon--type-theme($button-type-tokens) {
+  @include exports('button') {
+    @include button;
```

Comment:
One of the things about this RFC that makes me feel uncomfortable is that we're opening up the door for zoning. I'm struggling to think of a use case that can't be solved in a better way. It just feels like this is opening up a can of worms more than it's solving a problem. Talking about it with the designers there is also a level of hesitation with this, and I find it hard to believe something like governance will solve the issue of having the same component on the same page/experience in one or more themes.

<img src="https://user-images.githubusercontent.com/3793636/62346097-77e62900-b4ba-11e9-94c7-fb7988a06aa2.png" width="270px" />

I wonder if there is a way to solve this with a global switch or import like we've talked about in the past where it's an all or nothing approach.

```scss
@import '@carbon/dotcom-theme';
```

```scss
$type-theme: 'expressive';
```

Reply:
One last thing to consider about this RFC is that it feels like we might be unintentionally discouraging a unified design system. Instead of going to carbon for the one component, our adopters will now have to decide between two components y'alls and ours. Instead of unifying it's almost like we're continuing to divide. Just something to think about and consider as we decide to move forward.

```
@carbon/accordion
@carbon/dotcom-accordion

@carbon/button
@carbon/dotcom-button

@carbon/text-input
@carbon/dotcom-text-input
```